Android/Shared/ITMCopyAssets.gradle

```diff
@@ -0,0 +1,34 @@
+import java.nio.file.Files
+import java.nio.file.Paths
+import java.nio.file.StandardCopyOption
+
+def copyDirectory(srcDir, destDir) {
+    if (!Files.exists(srcDir))
+        throw new IOException("${srcDir.toAbsolutePath()} does not exist. Make sure react-app has been built prior to building the Android project.")
+    if (!Files.exists(destDir))
+        Files.createDirectories(destDir)
+    Files.walk(srcDir).forEach(srcPath -> {
+        def destinationPath = Paths.get(destDir.toString(), srcDir.relativize(srcPath).toString());
+        // skip copying directories that already exist
+        if (!Files.exists(destinationPath) || !Files.isDirectory(destinationPath))
+            Files.copy(srcPath, destinationPath, StandardCopyOption.REPLACE_EXISTING)
+    })
+}
+
+task copyITMAssets {
+    def reactAppDir = "${projectDir}/../../cross-platform/react-app"
+    println "reactAppDir ${new File(reactAppDir).canonicalPath}"
+    def assetsDir = "${projectDir}/../iTwinStarter/app/src/main/assets/ITMApplication"
```

Comment:
Isn't ${projectDir} iTwinStarter? (In other words, can't you remove ../iTwinStarter/ from this path and still have it be the same?

Reply:
The ProjectDir is the directory containing the build script (Android/Shared)